app/assets/javascripts/oxalis/model/bucket_data_handling/bucket_picker_strategies/flight_bucket_picker.js

```diff
@@ -149,12 +147,13 @@ export default function determineBucketsForFlight(
   const fallbackBuckets = isFallbackAvailable ? traverseFallbackBBox(getBBox(planeBuckets)) : [];
   traversedBuckets = traversedBuckets.concat(fallbackBuckets);
 
+  let currentCount = 0;
```

Comment:
The `const uniqueBucketMap = new ThreeDMap();` is used for the other two bucket pickers but not this one, is there a reason for that?

Reply:
"Historical growth" :see_no_evil: I forgot about the approach in the flight bucket picker (which was there before) and used the ThreeDMap instead. I'll probably consolidate this when doing the arbitrary-fallback-rendering PR.